src/tools/ScenarioMeasurement/Startup/Startup.csproj

```diff
@@ -18,4 +25,17 @@
     <ProjectReference Include="..\..\Reporting\Reporting\Reporting.csproj" />
   </ItemGroup>
 
+  <Target Name="DownloadPerfCollect" BeforeTargets="CoreCompile">
+    <PropertyGroup>
+      <PerfcollectUrl>https://aka.ms/perfcollect</PerfcollectUrl>
+    </PropertyGroup>
+    <DownloadFile SourceUrl="$(PerfcollectUrl)" 
+                  DestinationFolder="$(MSBuildProjectDirectory)"
+                  Retries="3"
+                  SkipUnchangedFiles="true">
+      <Output TaskParameter="DownloadedFile" ItemName="Content"/>
+    </DownloadFile>
+    <Copy SourceFiles="perfcollect" DestinationFolder="$(PublishDir)" />
+    <Copy SourceFiles="perfcollect" DestinationFolder="$(OutDir)" />
+  </Target>
```

Comment:
Can we also make this target dependent on non-Windows?

Reply:
I was worrying the publish won't always happen on Linux platform for Linux runs, so I didn't set the condition. But it seems the script task that runs the publish is platform-dependent already.